client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  return (
+    <div className="container">
+      <a className={ scss.backLink }
+        onClick={ (e) => handleClick(e, "") }
+        href={ props.initialPath } >
+        ‹ { props.t('back_to_todo') }
+      </a>
+
+      <h2 className={ scss.title } >{ props.t('title') }</h2>
+
+      <p className={ scss.description } >
+        { props.t('description') }
+      </p>
+
+      <div className={ scss.sloganImageContainer } >
+        <div className={ scss.sloganImage } ></div>
+      </div>
+
+      <div className={ scss.infoTextContainer } >
+        <div className={ scss.infoTextIcon } >
+          <i className="ss-info"></i>
```

Comment:
You **can not** use "ss-" icon classes directly. They will not work in OS installation. You need to use the IconMap and figure out the proper class, which may be ss or font-awesome.